install.sh

```diff
@@ -69,7 +69,7 @@ install_nvm_from_git() {
   if [ -d "$INSTALL_DIR/.git" ]; then
     echo "=> nvm is already installed in $INSTALL_DIR, trying to update using git"
     printf "\r=> "
-    cd "$INSTALL_DIR" && (command git fetch 2> /dev/null || {
+    pushd "$INSTALL_DIR" && (command git fetch 2> /dev/null || {
       echo >&2 "Failed to update nvm, run 'git fetch' in $INSTALL_DIR yourself." && exit 1
```

Comment:
I think I like that better, *provided* that those commands were supported in an old enough version of git that it won't suddenly raise the threshold for required git support.